avocado/utils/dmesg.py

```diff
@@ -115,3 +122,19 @@ def collect_errors_by_level(output_file=None, level_check=5):
             err += " Please check  dmesg log in debug log."
             LOGGER.debug(dmsg_log)
         raise DmesgError("Test is failed {}".format(err))
+
+
+def skip_dmesg_error(dmesg_error, skip_dmesg):
+    """remove dmesg error which want to skip.
+
+    :dmesg_error: dmesg messages
```

Comment:
If this is a string, I' m assuming this is a single dmesg line. Did I understood right?

Reply:
If not , we should choose better names here.

Reply:
not single line,its multiple line of dmesg.